frontend/public/components/utils/simple-tab-nav.tsx

```diff
@@ -31,9 +35,9 @@ export const SimpleTabNav: React.FC<SimpleTabNavProps> = ({
     <div>
       <Tabs
         onSelect={handleTabClick}
-        className={additionalClassNames}
+        className={classnames({ 'pf-v6-u-mb-lg': withinSidebar }, additionalClassNames)}
```

Comment:
```suggestion
        className={classnames({ 'pf-v6-u-mb-md': withinSidebar }, additionalClassNames)}
```
Better matches the forthcoming padding changes I'm working on.